float32 xDesired
float32 yDesired
float32 thetaDesired
bool 	commandDesired
